File: Programmers/level2/intactSquare/intact_square.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/62048

from math import gcd


def solution(w, h):
    print(w * h - (w + h - gcd(w, h)))
    return w * h - (w + h - gcd(w, h))

File: Programmers/level2/intactSquare/intactSquare.js
function solution(w, h) {
    let gcd = 1;
    let min = Math.min(w,h);
    
    for(let i = min; i > 0; i--){
        if(w % i === 0 && h % i === 0){
            gcd = i
            break;
        }
    }
    return w * h - (w + h - gcd)
}